dags/databricks-ml-example.py

```diff
@@ -0,0 +1,85 @@
+from datetime import datetime
+import logging
+
+from airflow.decorators import task, dag
+from airflow.providers.databricks.hooks.databricks import DatabricksHook
+from airflow.providers.databricks.operators.databricks import DatabricksSubmitRunOperator
+# from astronomer.providers.databricks.operators.databricks import DatabricksSubmitRunOperatorAsync
+
+import mlflow
+
+docs = """
+Demonstrates orchestrating ML pipelines executed on Databricks with Airflow
+"""
+
+@dag(
+    start_date=datetime(2022, 1, 1),
+    schedule_interval=None,
+    catchup=False,
+    doc_md=docs
+)
+def databricks_ml_example():
+
+    ingest_notebook = {
+        'notebook_path': '/Users/<your username>/<BigQuery-to-Databricks-notebook>',
+    }
+
+    ingest = DatabricksSubmitRunOperator(
+        task_id='ingest_notebook_task',
+        existing_cluster_id='0224-221140-suj0ngd4',
+        notebook_task=ingest_notebook
+        )
+ 
+    feauture_engineering_notebook = {
+        'notebook_path': '/Users/<your username>/<feauture-eng-notebook>',
+    }
+
+    feauture_engineering = DatabricksSubmitRunOperator(
+        task_id='feature_engineering_notebook_task',
+        existing_cluster_id='0224-221140-suj0ngd4',
+        notebook_task=feauture_engineering_notebook
+        )
+
+
+    train_notebook = {
+        'notebook_path': '/Users/<your username>/databricks_automl/<model-traning-notebook>'
+    }
+
+    train = DatabricksSubmitRunOperator(
+        task_id='train_notebook_task',
+        existing_cluster_id='0224-221140-suj0ngd4',
+        notebook_task=train_notebook,
+        do_xcom_push=True
```

Comment:
Specifying this isn't necessarily required since this is the default for this operator and the `BaseOperator` too. No strong opinion though.

Reply:
I tested without `do_xcom_push=True` and it failed to push anything.

Reply:
Oh I see what happened. The default value for `do_xcom_push` is True on the `main` branch in Airflow but not in a released Databricks provider yet which is wildly unfortunate. Not sure why that would have ever been changed from the `BaseOperator`. You're right, my fault. I was just looking at the `main` branch.